public/index.html

```diff
@@ -27,6 +27,31 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <title>Brain Defrost</title>
+
```

Comment:
@EthanDuvall Wanted to draw your attention to this file just so you could see what I added to attempt the ghpages routing fix. This was provided after doing some digging through the React docs and Github community posts. If it doesn't work, let's remove it!